app/Console/Commands/BulkReviewPosts.php

```diff
@@ -61,30 +61,30 @@ public function handle()
         $log = $this->option('log');
         $tags = $this->option('tag') ?? null;
 
-        $posts = Post::setEagerLoads([])
-            ->where('campaign_id', $this->argument('campaign'))
-            ->where('status', $this->argument('oldStatus'))
-            ->where('type', $this->argument('type'))
-            ->get();
+        Post::where([
+            ['campaign_id', $this->argument('campaign')],
```

Comment:
This is subjective, but I think the way you had the query before (as chained methods) reads a lil' nicer.

Reply:
@DFurnes totally ! I just wasn't sure the syntax for using `chunk` with the chained methods?

Reply:
Same, just instead of using `->get()` to grab the entire result set, call `->chunk(...)` to chunk it!